DataManagementSystem/Client/DataManager.py

```diff
@@ -947,14 +950,17 @@ def registerReplica( self, replicaTuple, catalog = '' ):
 
         'replicaTuple' is a tuple or list of tuples of the form (lfn,pfn,se)
     """
-    if isinstance( replicaTuple, list ):
-      replicaTuples = replicaTuple
-    elif isinstance( replicaTuple, tuple ):
-      replicaTuples = [ replicaTuple ]
+    if isinstance( replicaTuple, list ) or isinstance( replicaTuple, dict ) or isinstance( replicaTuple, set ):
```

Comment:
you can do isinstance( fileTuple, (list, dict, set) )
By the way, what about tuple, that is what if fileTuple is a tuple of tuple(s))